src/actions/TrackingActions.ts

```diff
@@ -33,6 +34,31 @@ export function trackConversion(
   }
 }
 
+export async function trackConversionWithReferral(
+  event: TrackingEventName,
+  opts: {
+    currencyCode: string
+    exchangeAmount: number
```

Comment:
While we're at it, can we add these to the tracking event instead of
just the dollarValue? 

At least for currencyCode, I can easily think of a hypothetical experiment where we are trying to
improve token UI/UX, and using measured token transactions as one of the metrics
of success.